packages/openapi-v3/src/controller-spec.ts

```diff
@@ -271,31 +256,62 @@ function processSchemaExtensions(
   }
 }
 
+function processSchemaExtensionsForRequestBody(
```

Comment:
How is `processSchemaExtensionsForRequestBody` different from the current `processSchemaExtensions` function?

I am reluctant to have different levels of support for schema extension depending on where the schema is coming from (`@param` vs. `@requestBody`). 

Can we find a way how leverage `processSchemaExtensions` for the new decorator syntax?

Reply:
> How is processSchemaExtensionsForRequestBody different from the current processSchemaExtensions function?

Reverted these change.